integreat_cms/cms/views/regions/region_list_view.py

```diff
@@ -55,7 +55,6 @@ def get(self, request, *args, **kwargs):
             regions = regions.filter(pk__in=region_keys)
 
         chunk_size = int(request.GET.get("size", settings.PER_PAGE))
-        # for consistent pagination querysets should be ordered
         paginator = Paginator(regions, chunk_size)
```

Comment:
I think it's a bad idea to convert this queryset to a list manually - if this would be more efficient in all scenarios, I think the Paginator class would do this itself.
The problem is, that calling `list()` instantly forces the queryset to evaluate, before it's even passed to the paginator. So if there are 1000 objects in the database, and the `chunk_size` is 10, passing the queryset as list would mean to retrieve all 1000 objects from the database, and then throwing 990 objects away to get the current page. In contrast, when we pass a queryset, it can do one query to determine that there are 1000 objects and then it can do a second query to retrieve only the 10 objects it needs. So even if we'll probably never have 1000 regions, I think it makes sense to keep the current behaviour and do an additional `COUNT()` for every pagination.

Reply:
Of course you are right!